packages/config/src/config.js

```diff
@@ -142,7 +144,24 @@ module.exports = ({
             inline: true,
             disableHostCheck: true,
             historyApiFallback: true,
-            writeToDisk: true
+            writeToDisk: true,
+            proxy: enableBetaRewrite ? {
+                'https://ci.foo.redhat.com:1337/beta': {
```

Comment:
What about qa and prod? I think it would be more reasonable to pass in `betaEnv` which would default to `ci` and if it is defined use this proxy.

Reply:
Forgot about the rest